### PR TITLE
2 DSC cards + 6 schemes

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/dark_wings_bring_your_downfall.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Dark Wings Bring Your Downfall
+ManaCost:no cost
+Types:Ongoing Scheme
+T:Mode$ AttackersDeclared | AttackingPlayer$ You | Execute$ TrigToken | TriggerZones$ Command | TriggerDescription$ Whenever you attack, create a 5/5 black Demon creature token with flying that's tapped and attacking.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ b_5_5_demon_flying | TokenOwner$ You | TokenTapped$ True | TokenAttacking$ True
+T:Mode$ Phase | Phase$ End of Turn | CheckSVar$ X | SVarCompare$ GE2 | TriggerZones$ Command | Execute$ TrigAbandon | OptionalDecider$ You | TriggerDescription$ At the beginning of each end step, if two or more creatures died under your control this turn, abandon this scheme.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/dark_wings_bring_your_downfall.txt:6]: Not optional

File: forge-gui/res/cardsfolder/upcoming/ursine_monstrosity.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Ursine Monstrosity
+ManaCost:2 G
+Types:Creature Bear Mutant
+PT:3/3
+K:Trample
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | Execute$ TrigMill | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of combat on your turn, mill a card and choose an opponent at random. CARDNAME attacks that player this combat if able. Until end of turn, CARDNAME gains indestructible and gets +1/+1 for each card type among cards in your graveyard.
+SVar:TrigMill:DB$ Mill | Defined$ You | NumCards$ 1 | SubAbility$ DBChoose
+SVar:DBChoose:DB$ ChoosePlayer | Defined$ You | Choices$ Player.Opponent | Random$ True | SubAbility$ DBAnimate
+SVar:DBAnimate:DB$ Animate | Defined$ Self | staticAbilities$ AttackChosen | Duration$ UntilEndOfCombat | SubAbility$ DBPump
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ursine_monstrosity.txt:9]: use effect, the ability isn't gained

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/ursine_monstrosity.txt:9]: Oh, then Ruhan of the Fomori is wrong...

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/ursine_monstrosity.txt:9]: Knight Rampager

**Review by tool4ever — APPROVED** [review]

**Pull request merged**